setup.py

```diff
@@ -16,6 +16,7 @@
     'itsdangerous',
     'requests',
     'PyJWT',
+    'django-crispy-forms',
```

Comment:
Wat voor verplichtingen (qua settings) brengt dat met zich mee? Dit komt dan in efcis, trs, sentry en nxt terecht.